tools/unitctl/unitctl/src/unitctl.rs

```diff
@@ -14,7 +14,8 @@ pub(crate) struct UnitCtl {
         short = 's',
         long = "control-socket-address",
         value_parser = parse_control_socket_address,
-        help = "Path (unix:/var/run/unit/control.sock), tcp address with port (127.0.0.1:80), or URL"
+        id = "CONTROL_SOCKET_ADDRESS",
```

Comment:
I think we want `value_name` not `id` here, but please verify

Reply:
yep, you're right. Value name did not come up on https://docs.rs/clap/latest/clap/_derive/index.html#arg-attributes

Reply:
@avahahn Does the suggested change look good to you?

Reply:
@javorszky From the first docs link you posted

> Raw attributes: Any [Arg method](https://docs.rs/clap/latest/clap/struct.Arg.html) can also be used as an attribute

So you can just use `value_name = "CONTROL_SOCKET_ADDRESS"` and everything will be great.

Reply:
OK, I'll apply this change and if the tests past, merge the sucker...

Reply:
Hmm, looks like it was already changed...